docs/_documentations/intellij-firstproject.md

```diff
@@ -35,14 +35,13 @@ If you have an IntelliJ project already open, you can create more projects.
 
 Congratulations, you have now created, built and run your first Codewind project using Codewind on your local machine.
 
-# Next Steps
+## Next Steps
 
-For next steps, choose from the following:
+Choose from the following:
 
-* If you intend to use Codewind [remotely](./remote-codewind-overview.html), you have now completed the pre-requisite step of installing Codewind on your local machine and can proceed to:
+**Getting started with Codewind - next steps** Continue to instructions for [Making a code change with IntelliJ](intellij-codechange.html) to see the auto-build-and-run feature in action.
```

Comment:
The link should be, `(./intellij-codechange.html)`. I think it needs the `./` part to work.